### DISCUSSION TITLE
How to enable Photoswipe?

### DISCUSSION THREAD

**Discussion by foxdie987** [CaiJimmy/hugo-theme-stack#634] [Q&A]
Is Photoswipe enabled by default, or do I need to edit a parameter in `config.yaml`, or enable it in Front Matter field?

Because I don't see Lightbox Javascript effect when using `figure` shortcode, or `![]()` markdown.

Thanks in advance !

**Comment by CaiJimmy** [comment]
PhotoSwipe only works with images inserted using the page bundle feature. No configuration is needed.

**Reply by foxdie987** [reply]
Thank you very much, it's okay now. Before, I stored my images under `static` folder so Photoswipe didn't work for me.

After that, I keep my images all in the same folder bundled with my markdown file, and now it works with markdown syntax: `![Image alt text](image1.jpg)`
```
├── posts
│   ├── my_post_1
│   │   ├── index.md
│   │   ├── image1.jpg
│   │   ├── image2.jpg
│   │   └── .....
```

This format is also working, you may need a headless `_index.md` in image folder though. The following markdown syntax will work: `![Image alt text](images/image1.jpg)`
```
content/
├── posts
│   ├── my_post1
│   │   ├── image
│   │   │       ├── image1.jpg
│   │   │       ├── image2.jpg
│   │   └── index.md
```

---
But the `figure` shortcode will not work with Photoswipe, even with page bundles. I don't know why.

By the way, I'm using this below shortcode to get a lot of photos from my picture folder to avoid repeating use of `![Image alt text](image_number.jpg)`. The code loads the pictures properly, but it can't interact with Photoswipe. Can you show me how to fix this? Thank you. 

```
{{ $resources := .Page.Resources.ByType "image"}}
  {{ range $resources }}
    {{ with . }}
      <img style="max-width: 100%;" src="{{ .RelPermalink }}">
      <br />
    {{ end }}
  {{ end }}
```
